daemon/userconsent.go

```diff
@@ -75,7 +75,13 @@ func NewConsentChecker(
 //   - save consent as completed and accepted, no consent flow for standard mode countries
 func (acc *AnalyticsConsentChecker) PrepareDaemonIfConsentNotCompleted() {
 	if acc.IsConsentFlowCompleted() {
-		// nothing to do
+		var cfg config.Config
+		if err := acc.cm.Load(&cfg); err != nil {
+			log.Println(internal.ErrorPrefix, "failed to load config while running daemon preparation", err)
+		}
+		if err := acc.analytics.Init(cfg.AnalyticsConsent); err != nil {
```

Comment:
Maybe we should make `Init` more explicit in case of config load fails? Or maybe just load the config earlier and and pass it to `IsConsentFlowCompleted` and then use here for `Init` 🤔